stores/github/authorization-model.json

```diff
@@ -151,6 +216,69 @@
         "repo_writer": {
           "this": {}
         }
+      },
+      "metadata": {
+        "relations": {
+          "member": {
+            "directly_related_user_types": [
+              {
+                "type": "organization"
+              }
+            ]
+          },
+          "owner": {
+            "directly_related_user_types": [
+              {
+                "type": "user"
```

Comment:
should owner be `organization`?

Reply:
I believe one or more users own an org. I don't think orgs own an org, but I may be wrong.

Reply:
Yep - an org owns a repo, a user owns an org